src/components/WalletModal/Option.tsx

```diff
@@ -17,26 +24,17 @@ const OptionCardLeft = styled.div`
 `
 
 const OptionCardClickable = styled.button<{ selected: boolean }>`
-  background-color: ${({ theme }) => theme.backgroundModule};
+  align-items: center;
+  background-color: unset;
   border: none;
-  width: 100% !important;
-
+  cursor: pointer;
   display: flex;
+  flex: 1 1 auto;
   flex-direction: row;
-  align-items: center;
   justify-content: space-between;
+  opacity: ${({ disabled, selected }) => (disabled && !selected ? '0.5' : '1')};
   padding: 18px;
-
   transition: ${({ theme }) => theme.transition.duration.fast};
-  opacity: ${({ disabled, selected }) => (disabled && !selected ? '0.5' : '1')};
-
-  &:hover {
-    cursor: ${({ disabled }) => !disabled && 'pointer'};
-    background-color: ${({ theme, disabled }) => !disabled && theme.hoverState};
-  }
-  &:focus {
-    background-color: ${({ theme, disabled }) => !disabled && theme.hoverState};
-  }
```

Comment:
2 things happening here:

1. I alphabetized some scattered properties
2. I removed the hover/focus rules and bumped them up to a parent element so the new dom structure would retain the existing behavior.

Reply:
thanks for this comment!